src/Element/EntityAutocompleteMultiple.php

```diff
@@ -0,0 +1,278 @@
+      $element[$i]['target'] = [
+        '#type' => 'entity_autocomplete',
+        '#title' => t('@title (value @number)', ['@title' => $element['#title'], '@number' => $i + 1]),
+        '#title_display' => 'invisible',
+        '#target_type' => $element['#target_type'],
+        '#selection_handler' => $element['#selection_handler'],
+        '#selection_settings' => $element['#selection_settings'],
+        '#validate_reference' => FALSE,
+        '#maxlength' => $element['#maxlength'],
+        '#default_value' => $element['#default_value'][$i] ?? NULL,
+        '#size' => $element['#size'],
+        '#placeholder' => $element['#placeholder'],
+      ];
+      $element[$i]['_weight'] = [
+        '#type' => 'weight',
+        '#title' => t('Weight for row @number', ['@number' => $i + 1]),
+        '#title_display' => 'invisible',
+        '#default_value' => $i,
+        '#weight' => 100,
+      ];
+    }
+
+    $id_prefix = implode('-', $parents);
+    $wrapper_id = Html::getUniqueId($id_prefix . '-add-more-wrapper');
+    $element['#prefix'] = '<div id="' . $wrapper_id . '">';
+    $element['#suffix'] = '</div>';
+    $element['add_more'] = [
+      '#type' => 'submit',
+      '#name' => strtr($id_prefix, '-', '_') . '_add_more',
+      '#value' => $element['#add_more_title'],
+      '#attributes' => ['class' => ['field-add-more-submit']],
+      '#limit_validation_errors' => [$element['#array_parents']],
+      '#submit' => [[get_called_class(), 'addMoreSubmit']],
+      '#ajax' => [
+        'callback' => [get_called_class(), 'addMoreAjax'],
+        'wrapper' => $wrapper_id,
+        'effect' => 'fade',
+      ],
+    ];
+    // Remove leftover from root element array.
+    unset($element['#maxlength']);
+    return $element;
+  }
+
+  /**
+   * Validate autocomplete field values.
+   *
+   * @param array $element
+   *   The element.
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The form state.
+   * @param array $complete_form
+   *   The original complete form array.
+   */
+  public static function validateEntityAutocompleteItems(array &$element, FormStateInterface $form_state, array &$complete_form): void {
+    // We do not need to use #value element for avoiding double work
+    // related to handling values for entity_autocomplete form elements.
+    // Instead, we can just slightly cleanup and extract needed data from
+    // ready form state values.
+    $values = $form_state->getValue(implode('.', $element['#parents'])) ?? [];
+    unset($values['add_more']);
+    $values = array_filter($values, function ($value) {
+      return !empty($value['target']);
+    });
+    usort($values, function ($a, $b) {
+      return SortArray::sortByKeyInt($a, $b, '_weight');
+    });
+    $entity_ids = array_map(function (array $item) {
+      return $item['target'];
+    }, $values);
+
+    if (empty($entity_ids)) {
+      $form_state->setError($element, t('You have to select at least 1 content owner.'));
```

Comment:
Don't hardcode things here please ('content owner') as will move this out of the module.

Also, add the String translation trait and use $this-t()

Reply:
I have changed to 'You have to select at least 1 entity reference.'
It is static method so we cannot use $this->t()